### PR TITLE
Overview Updates

### PR DESCRIPTION
**Pull request by carolinerobbins**
- Added zoom image (still needs tweaking on positioning)
- Moved details into state for optimization
- Added tests to achieve 70% coverage
- Fixed merge error in App.js
- Fixed details reducer error (forgot to import action)

### CODE REVIEW

File: src/components/App.jsx
Comment (pdaly91) [inline src/components/App.jsx:1]: I noticed you removed answer on line 20 and added back the button on line 34. Is that correct? Wanted to double check first

Comment (carolinerobbins) [reply src/components/App.jsx:1]: Oops I did not merge that correctly - I meant to combine!

File: src/components/overview/Gallery.jsx
Comment (pdaly91) [inline src/components/overview/Gallery.jsx:1]: You have some duplicated lines in here, could you please check?

File: src/components/App.jsx
Comment (pdaly91) [inline src/components/App.jsx:1]: You have some duplicated lines in here

File: src/components/overview/Styles.jsx
Comment (pdaly91) [inline src/components/overview/Styles.jsx:1]: Duplicate lines

File: src/components/relatedProducts/ComparisonModal.jsx
Comment (pdaly91) [inline src/components/relatedProducts/ComparisonModal.jsx:1]: Maybe some duplicates here too?

**Review by pdaly91 — CHANGES_REQUESTED** [review]

**Review by ahusu — APPROVED** [review]
Looks good overall, I think you gotta check out Patrick's concerns first though

File: src/components/overview/Details.jsx [hunk 1]
```diff
@@ -58,65 +62,6 @@ export default function Details() {
         )}
         <Styles />
       </div>
-      <div className="flex-2 w-2/3 mt-4 border-r-2 border-black">
```
[the remark this answers is not in the document]
Comment (carolinerobbins) [reply src/components/overview/Details.jsx:61]: Yes, I needed to separate it so the slogan/description show under the gallery!

File: src/components/App.jsx
Comment (carolinerobbins) [reply src/components/App.jsx:1]: resolved!

File: src/components/overview/Gallery.jsx
Comment (carolinerobbins) [reply src/components/overview/Gallery.jsx:1]: resolved!

**Review by pdaly91 — APPROVED** [review]
Thanks for fixing those things!

**Pull request merged**